src/components/screens/Sessions.tsx

```diff
@@ -28,6 +31,7 @@ export default function Sessions() {
   const isEmptyState = sessionsLength === 0;
   const hasProposals = pendingProposals.data.length > 0;
   const analytics = useAnalytics();
+  const { lastConnectionApps } = useRecentConnection();
```

Comment:
We can use the `useAtomValue` directly
```suggestion
  const lastConnectionApps = useAtomValue(sortedRecentConnectionApps);
```